frontend/src/components/FacilityBooking/ViewBookingCard.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { Dispatch, SetStateAction, useEffect, useState } from 'react'
+import styled from 'styled-components'
+
+import ViewBookingCardButton from '../../assets/viewBookingCardButton.svg'
+import ViewBookingCardUserIcon from '../../assets/viewBookingUserIcon.svg'
+
+import { Booking } from '../../store/facilityBooking/types'
+import { unixToFullDate } from '../../common/unixToFullDate'
+import { unixTo12HourTime } from '../../common/unixTo12HourTime'
+import { unixToFullDay } from '../../common/unixToFullDay'
+import { DOMAIN_URL, ENDPOINTS } from '../../store/endpoints'
+
+const BackgroundOverlay = styled.div`
+  position: fixed;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.7);
+  z-index: 202;
+  cursor: pointer;
+`
+
+const BookingContainer = styled.div`
+  position: absolute;
+  padding-bottom: 20px;
+  top: 80px;
+  width: 85%;
+  max-width: 400px;
+  cursor: pointer;
+  background-color: #ffffff;
+  border-radius: 20px;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  z-index: 205;
+`
+
+const BookingHeader = styled.div`
+  width: 100%;
+  height: 20%;
+  cursor: pointer;
+  background-color: #f1f3f7;
+  border-radius: 20px 20px 0px 0px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  z-index: 2;
+  text-align: center;
+  padding: 20px 0px;
+`
+
+const EventName = styled.div`
+  font-size: 22px;
+  font-weight: 700;
+  margin: 0;
+`
+
+const BookingCca = styled.div`
```

Comment:
can afford to use p since this is solely text too
```suggestion
const BookingCca = styled.p`
```

Reply:
9155384d7